.buildkite/pipeline.yml

```diff
@@ -168,3 +166,41 @@ steps:
     plugins:
       - junit-annotate#v2.4.1:
           artifacts: build/TEST-go-integration*.xml
+
+  - group: "Desktop Windows tests"
+    key: "extended-windows"
+    steps:
+      - label: "Unit tests - Windows 10"
+        key: "unit-tests-win10"
+        branches: "main v7.* 7.* 8.* v8.*"
+        command: ".\\.buildkite\\scripts\\steps\\unit-tests.ps1"
+        artifact_paths:
+          - "build/TEST-**"
+          - "build/diagnostics/*"
+          - "coverage.out"
+        agents:
+          provider: "gcp"
+          # TODO create own image
```

Comment:
Is this TODO relevant? Why do we need to create our own image ?

Reply:
Yes. We plan to create our own VM images to reinstall more tools and reduce the build time